### PR TITLE
Account payment details

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA: ask me.

File: jujugui/static/gui/src/app/components/account/account.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /*
 This file is part of the Juju GUI, which lets users view and manage Juju
 environments within a graphical interface (https://launchpad.net/juju-gui).
-Copyright (C) 2015 Canonical Ltd.
+Copyright (C) 2016 Canonical Ltd.
```
Comment (kadams54) [inline jujugui/static/gui/src/app/components/account/account.js]: Why not 2017?

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6912/

**Comment by hatched** [comment]
:shipit: Thanks @huwshimi

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**